make test-e2e-docker fails

**Describe the bug**

`scripts/test_e2e.sh` is required for this make target and that file does not exist


**Error Message**

<pre>
```console
docker-compose -p athense2e up --exit-code-from=teste2e teste2e
WARN[0000] /home/matt/Projects/athens/docker-compose.yml: `version` is obsolete 
Attaching to teste2e-1
Error response from daemon: failed to create task for container: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: exec: "./scripts/test_e2e.sh": stat ./scripts/test_e2e.sh: no such file or directory: unknown
make: *** [Makefile:82: test-e2e-docker] Error 1
```
</pre>


**To Reproduce**
Steps to reproduce the behavior:

1. Run `make test-e2e-docker`


**Expected behavior**

E2E tests run in Docker Compose


**Environment (please complete the following information):**
 - OS: Linux 64bit
 - Go version : 1.21
 - Proxy version : v0.13.3
 - Storage (fs/mongodb/s3 etc.) : fs


**Additional context**
N/A

--- a/Makefile
+++ b/Makefile
@@ -74,7 +74,7 @@ test-unit-docker: ## run unit tests with docker
 
 .PHONY: test-e2e
 test-e2e:
-	cd e2etests && go test --tags e2etests
+	./scripts/test_e2e.sh
 
 .PHONY: test-e2e-docker
 test-e2e-docker:

--- a/scripts/test_e2e.sh
+++ b/scripts/test_e2e.sh
@@ -1,0 +1,7 @@
+#!/bin/bash
+
+# test_e2e.sh
+
+# Run the e2e tests with the race detector enabled
+set -xeuo pipefail
+cd e2etests && go test --tags e2etests -race

